big-data-utils/src/main/java/edu/illinois/cs/cogcomp/bigdata/lucene/Lucene.java

```diff
@@ -267,9 +266,9 @@ public static QueryParser newQueryParser(String string) {
 	 * @param id
 	 * @return
 	 */
-	public static Query intQuery(String field, int id) {
-		return NumericRangeQuery.newIntRange(field, id, id, true, true);
-	}
+//	public static Query intQuery(String field, int id) {
```

Comment:
Lucene has removed this kind of queries from the API. All queries are to be made using the QueryParser now.